api/jobset/v1alpha2/jobset_types.go

```diff
@@ -62,6 +62,24 @@ const (
 	JobSetStartupPolicyCompleted JobSetConditionType = "StartupPolicyCompleted"
 )
 
+// the status of JobSet
+const (
+	// Succeed means the JobSet execution is successful.
```

Comment:
We should use the conditions: https://github.com/kubernetes-sigs/jobset/blob/main/api/jobset/v1alpha2/jobset_types.go#L52

I think phase should take one of four values: completed, failed and suspended and if none of those are true, then set to running

Reply:
I followed the review's suggestion and set the phase to Running by default. Otherwise, the phase is changed according to the conditions.